.github/workflows/e2e-nightly-master.yml

```diff
@@ -6,6 +6,13 @@
 name: e2e-nightly-master
 on:
   workflow_dispatch:  # allow running workflow manually
+    inputs:
+      notifySlackOnFailure:
+        type: boolean
+        default: true 
+      notifySlackOnSuccess:
+        type: boolean
+        default: true 
```

Comment:
This looks probably fine (not an expert) although I can't _reeeeeaally_ imagine that we want to actually notify in slack for manual runs?

Reply:
The reason why it's there because this doubles as the e2e nightly masters run which we want to notify in slack. We could separate the manual and nightlies into different workflows